pandas/io/excel.py

```diff
@@ -295,6 +295,11 @@ def _parse_excel(self, sheetname=0, header=0, skiprows=None, skip_footer=0,
         if 'chunksize' in kwds:
             raise NotImplementedError("Reading an Excel file in chunks "
                                       "is not implemented")
+        if parse_dates:
+            raise NotImplementedError("parse_dates is not implemented")
```

Comment:
say parse_dates keyword to ExcelReader is not implemented (and similarly on other one)